### ISSUE TITLE
SideSwipe not working inside ScrollView

### ISSUE THREAD

**Issue by AlexisLeon** [kkemple/react-native-sideswipe#13]
Following your example: 

```javascript
import { View, Dimensions, ScrollView } from 'react-native';

export default class SweetCarousel extends Component {
  state = {
    currentIndex: 0,
  };

  render = () => {
    // center items on screen
    const { width } = Dimensions.get('window');
    const contentOffset = (width - Card.WIDTH) / 2;

    return (
      <ScrollView>
        <SideSwipe
          index={this.state.currentIndex}
          itemWidth={Card.WIDTH}
          style={{ width }}
          data={data}
          contentOffset={contentOffset}
          onIndexChange={index =>
            this.setState(() => ({ currentIndex: index }))
          }
          renderItem={({ itemIndex, currentIndex, item, animatedValue }) => (...)}
        />

        ...
      </ScrollView>
    );
  };
}
```

**Comment by kkemple** [comment]
hey! thanks for reporting this! looking into it now

**Comment by kkemple** [comment]
this should be fixed in 1.2.1, please let me know if you still have issues!

here is it working in a snack
![feb-07-2018 16-02-19](https://user-images.githubusercontent.com/3629876/35941498-65aa037c-0c20-11e8-8261-1f7f4d60c12b.gif)

**Comment by kkemple** [comment]
going to close this since no response
